docker/buildkite/docker-compose-local-async-wf.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.5"
+# version: "3.5"
```

Comment:
why is this line commented out?

Reply:
locally `docker-compose` was complaining about obsolete version. I'll revert. if new version of docker-compose doesn't need this we can remove from all files at bulk